javascript/modules/client/test/integration/client-addressList-modules/decoding.test.ts

```diff
@@ -0,0 +1,127 @@
+// @ts-ignore
+declare const describe, it, expect;
+
+import {
+  ClientAddressList,
```

Comment:
Could we also group tests by `component/<module>`?

Reply:
Isn't that how are grouped at this moment?
```
- integration/client-modules
| ->  integration/client-modules/encoding
| ->  integration/client-modules/decoding
| ->  integration/client-modules/estimation
| ->  integration/client-modules/methods

and so on
```
what do you suggest?

Reply:
My point is about the path names with dashes, I would have folder hierarchies grouping things more than `client-addressList-modules`. 

Low priority comment, it's just about testing ergonomics